structs/forks/bellatrix/bellatrix.go

```diff
@@ -25,6 +26,14 @@ func (b *SubmitBlockRequest) Validate() bool {
 		b.BellatrixExecutionPayload.EpTimestamp > 0
 }
 
+func (b *SubmitBlockRequest) Raw() []byte {
+	return b.BellatrixRaw
```

Comment:
You got me. I have no idea where you get this from : > I thought you would need to change api.go to have that available. Or extend unmarshal